tools/ci/manifest_build.py

```diff
@@ -134,9 +133,17 @@ def create_release(manifest_path, owner, repo, sha, tag, body):
         upload_resp = request(upload_url, "Manifest upload", data=upload_data, params=params,
                               headers={'Content-Type': 'application/octet-stream'})
         if not upload_resp:
-            success = False
+            return False
+
+    release_id = create_resp["id"]
+    edit_url = "https://api.github.com/repos/%s/%s/releases/%s" % (owner, repo, release_id)
+    edit_data = {"draft": False}
+    edit_resp = request(edit_url, "Release publishing", json_data=edit_data)
```

Comment:
It looks like this will do a `POST` request not `PATCH`?

Reply:
Yep, didn't spot the `PATCH` in https://developer.github.com/v3/repos/releases/#edit-a-release-asset and it looks like GitHub accepts this.

The easy way to fix this is to add a `method=None` argument to `request`, but unless it's a string enum it would leak that fact that `requests` is being used within.

Reply:
I think adding a `method` argument that defaults to `None` is fine.

Reply:
I've pushed a third commit to do just that, and verified that it could produce https://github.com/foolip/wpt/releases/tag/merge_pr_19493.